miner/worker.go

```diff
@@ -77,6 +78,13 @@ const (
 	staleThreshold = 7
 )
 
+var (
+	minerZeroTxBroadcastMetrics    = metrics.NewRegisteredTimer("chain/miner/zerotx/broadcast", nil)
```

Comment:
Why it is timer not histogram or gauge or counter?

Reply:
counter and gauge basically don't calculate time

Reply:
histogram I need to check but I think we can get that data from the timer itself on datadog I suppose

Reply:
So it's mostly about block time. Ok then.